### PR TITLE
Fix join section

### PR DESCRIPTION
**Pull request by sahsisunny**
### What is the change?

Following files are changed:

- `css/main.css` - Fix the description text and set text to center to anchor tag.
- `index.html` - Change button to anchor tag and add href(go to the info text section) and add id to the info text section.
- `js/index.js` - Remove unnecessary code of joinBtn button.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots
After :

Fix Join Section

After Request Changes

Fix Line Height

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -101,8 +101,3 @@ modalTriggers.forEach((trigger) => {
     });
   });
 });
-
-const joinBtn = document.getElementById('btn-join');
-joinBTN.addEventListener('click', () => {
-  window.location.href = 'https://realdevsquad.com/join';
-});
```
Comment (rohan09-raj) [inline js/index.js:108]: It's behind a feature flag, we can move ahead with the functionality of clicking the button
Check here :
https://realdevsquad.com/?join=true
https://realdevsquad.com

We only need to fix the functionality as mentioned in the issue, the click was not working

File: index.html [hunk 2]
```diff
@@ -258,7 +258,7 @@ <h1 class="features-title">How can you join?</h1>
             put in any efforts for yourself. We want to value everyone’s time
             and efforts.
           </p>
-          <button class="btn-join" id="btn-join">Join the Squad</button>
+          <a class="btn-join" href="#info-note">Join the Squad</a>
```
Comment (rohan09-raj) [inline index.html:261]: This is not visible in the production until user puts the feature flag, so we can move ahead with the button

**Review by rohan09-raj — CHANGES_REQUESTED** [review]
@sahsisunny Please look into UI, it still has too much whitespace between the elements and I think those two paragraphs can be combined to one as on desktop site, it looks odd to see this much line height.

File: js/index.js (hunk 1 above)
Comment (sahsisunny) [reply js/index.js:108]: done

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**